src/blog/kuadrant-v1.md

```diff
@@ -0,0 +1,52 @@
+
+### What's new in v1?
+
+There are too many features and bug fixes to mention (although you are welcome to peruse our [release changelogs](https://github.com/Kuadrant/kuadrant-operator/releases/tag/v1.0.0)). Here are some highlights:
+
+- **CRDs Graduation**: `AuthPolicy`, `DNSPolicy`, `RateLimitPolicy`, and `TLSPolicy` have now graduated to `v1` in their CRDs, signalling their readiness for production use.
+- **Policy Machinery Integration**: Our core components now implement [policy-machinery](https://github.com/Kuadrant/policy-machinery), providing a powerful way to visualise the "state of the world" for policies on clusters. No more guessing where policies and their behaviours originate from—policy machinery and visualisation through the console plugin make it easier than ever to reason about policy origins.
+- **Shift to the Sail Operator**: With v1, we recommend installing Istio using the new Istio [Sail Operator](https://istio.io/latest/blog/2024/introducing-sail-operator/) instead of `istioctl`.
+- **CEL Support**: [CEL](https://cel.dev/) (Common Expression Language) is now supported for [`AuthPolicy`](https://github.com/Kuadrant/authorino/blob/main/docs/features.md#common-feature-common-expression-language-cel) and [`RateLimitPolicy`](https://docs.kuadrant.io/1.0.x/kuadrant-operator/doc/reference/ratelimitpolicy/#predicate) predicates, enabling more flexible and expressive configurations.
+- **SectionName Support**: Added support for [SectionName](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SectionName), allowing users to apply policies to specific sections of Gateways.
+- **New Console Plugin**: For [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) / [OKD](https://okd.io/) console users, a new Console Plugin is available for managing policies. We plan to make some of this functionality available through other means for Kubernetes users in the near future.
+
+### Components in this release
+
+The individual components that make up this release of the Kuadrant Operator:
+
+| **Component**       | **Version**                                                                                       |
+|---------------------|---------------------------------------------------------------------------------------------------|
+| Authorino Operator  | [v0.16.0](https://github.com/Kuadrant/authorino-operator/releases/tag/v0.16.0)                    |
+| Limitador Operator  | [v0.12.1](https://github.com/Kuadrant/limitador-operator/releases/tag/v0.12.1)                    |
+| DNS Operator        | [v0.12.0](https://github.com/Kuadrant/dns-operator/releases/tag/v0.12.0)                          |
+| WASM Shim           | [v0.8.1](https://github.com/Kuadrant/wasm-shim/releases/tag/v0.8.1)                               |
+| Console Plugin      | [v0.0.14](https://github.com/Kuadrant/console-plugin/releases/tag/v0.0.14)                        |
```

Comment:
If none of these operators are 1.0, how is this the Kuadrant 1.0 release?

Reply:
This is a fair question. One we will no doubt debate further over the next few releases. The Kuadrant Opertor is v1 and its APIs are v1. These components are its dependencies that are also part of the kuadrant org. The v1 release signifies the kuadrant policy APIs  and the kuadrant operator as being stable (RateLimitPolicy, DNSPolicy, TLSPolicy and AuthPolicy) we wont break these APIs. If one of the dependencies has a change in their API, the kuadrant-opertor and the kuadrant community will be responsible for ensuring we handle that change without breaking the external v1 policy APIs.